### ISSUE TITLE
[Bug]: Accessability: unlabeled tabs

### ISSUE THREAD

**Issue by shortwavesurfer2009** [meshtastic/Meshtastic-Android#1554]
### Contact Details

this issue

### What happened?

The tabs are not accessible with the TalkBack screen meter.

### App Version

The tabs are not properly labeled for screen reader users, such as with TalkBack.

![Image](https://github.com/user-attachments/assets/5ce00ca2-2e32-41db-bc39-d56f8fdee1cd)

### Phone

OnePlus Nord N200
LineageOS 21 (Android 14)
App v2.5.15 (F-Droid)

### Device

None

### Firmware

Na

### Relevant log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by shortwavesurfer2009** [comment]
> Thanks for the a11y call-out! Please let us know if you see anything further that we can improve here

I tried the newest debug build from the actions and it's perfect. I did find a few more issues though that I didn't discover earlier.

https://github.com/meshtastic/Meshtastic-Android/issues/1632
